main.go

```diff
@@ -631,11 +649,33 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
 							export.CreateExcludeProfileNameSelectableCodeSignGroupFilter(defaultProfile.Name))
 						if len(filteredCodeSignGroups) > 0 {
 							codeSignGroups = filteredCodeSignGroups
+
+							log.Debugf("Groups after removing default profile:")
+							for _, group := range codeSignGroups {
+								log.Debugf(group.String())
+							}
 						}
 					}
 				}
 
-				iosCodeSignGroups := export.CreateIosCodeSignGroups(codeSignGroups)
+				var iosCodeSignGroups []export.IosCodeSignGroup
+
+				for _, selectable := range codeSignGroups {
+					bundleIDProfileMap := map[string]profileutil.ProvisioningProfileInfoModel{}
+					for bundleID, profiles := range selectable.BundleIDProfilesMap {
+						bundleIDProfileMap[bundleID] = profiles[0]
```

Comment:
Make sure there is a profile in the array before calling `profiles[0]`.
I've checked, but we don't check it before.